src/content/1.7/scale/_index.md

```diff
@@ -38,3 +38,14 @@ It may not be as simple as it sounds, as PrestaShop requires many files being sh
 Also, once you have set up your first server, setting additional ones should be far easier.
 
 The good thing is that, though requiring more sweat at first, horizontal scaling is better in terms of performances. Done well, adding a second server may easily double your application performance, if not more.
+
+### Further than LAMP
+
+Another way to scale your shop is to introduce other systems to improve your performances. Here is a quicklist of what can be used:
+
+1. Using a reverse proxy such as varnish for Front Office
```

Comment:
```suggestion
1. Using a HTTP accelerator such as varnish for Front Office
```
If people are looking for reverse proxy, it will not necessarily increase performance.